util/test/tests/D3D11/D3D11_Shader_Debug_Zoo.py

```diff
@@ -17,6 +17,8 @@ def check_capture(self):
         failed = False
 
         # Loop over every test
+        rdtest.log.print("Performing general tests:")
+        rdtest.log.indent()
```

Comment:
FYI - I added `rdtest.log.begin_section` and `rdtest.log.end_section`. It adds an extra collapsible level in the HTML view which is nice, as well as an indent.